add note on HTLM IAM for fire/androidtv (#3733)
braze-inc/braze-docs: _docs/_developer_guide/platform_wide/tv_and_ott.md

  - The priority must be set to "HIGH" for these to appear. All notifications appear in the Fire TV settings menu.
- Content Cards
- In-app messages

>   - HTML messages are not supported by Fire OS

Author: ```suggestion
  - HTML messages are not supported on Fire OS
```

>   - HTML messages are not supported on Fire OS

Author: ```suggestion
  - HTML messages are not supported on non-touch devices
```

- Push Notifications
- Content Cards
- In-app messages

>   - HTML messages are not supported by Fire OS

Author: ```suggestion
  - HTML messages are not supported on Fire OS
```

>   - HTML messages are not supported on Fire OS

Author: ```suggestion
```

- Data and Analytics collection for cross-channel engagement
- Content Cards
- In-app messages 

>   - HTML messages are not supported by Fire OS

Author: ```suggestion
  - HTML messages are not supported on Android TV
```

>   - HTML messages are not supported on Android TV

Author: ```suggestion
  - HTML messages are not supported on non-touch devices
```
